enzy_htp/structure/atom.py

```diff
@@ -233,15 +233,18 @@ def radius(self, method: str = "ionic") -> float:
         radius = chem.get_atom_radii(self.element, method)
         return radius
 
-    @dispatch
-    def distance_to(self, point: Atom) -> float:
-        """Get the distance to the other atom or a point."""
-        return mh.get_distance(self.coord, point.coord)
+    #@dispatch
+    #def distance_to(self, point: Atom) -> float:
+    #    """Get the distance to the other atom or a point."""
+    #    return mh.get_distance(self.coord, point.coord)
 
-    @dispatch
+    #@dispatch
```

Comment:
Did you make this from an error from plum that says "float" is not achieved. I dont really understand that error and it disappears after it tried it the 5th time. like wtf.

Reply:
yeah i don't understand it either. I'll check out the existing version of the file from `develop_refactor`

Reply:
It seems a problem in old plum-dispatch version. I have it when my version is 1.5.9 in an environment and it goes away when I update it to 1.6 or to 2.1.1 using `pip`

Reply:
fixed via a specific commit